docs/src/app/components/Master.js

```diff
@@ -22,15 +23,18 @@ class Master extends Component {
 
   static childContextTypes = {
     muiTheme: PropTypes.object,
+    uniqueIdGen: PropTypes.func.isRequired,
   };
 
   state = {
     navDrawerOpen: false,
+    uniqueIdGen: getUniqueIdGenerator(),
```

Comment:
Do we need it in the state? Can't we move it at the top of the file (outside of the component)?